### DISCUSSION TITLE
Rudia Rd6563F Support? (Fiber Laser)

### DISCUSSION THREAD

**Discussion by JacobWeinrich** [meerk40t/meerk40t#2620] [Q&A]
Hello im using a Rudia Rd6563. The controller normally uses RdCutist, and I want to use Lightburn now. With Lightburn, everything sort of works. The first issue is the Z-axis is not supported (This isn't a huge deal). My MAIN issue is halfway through a cut with light burn, and I will get an incompatible file error; only do this on larger files. My question is with this software can I use it to control my fiber laser with lightburn and not have this issue?

**Comment by jpirnay** [comment]
Let me try to rephrase my understanding: You do have a ruida controller and a fibre laser being controlled with it? To be honest our experience with Ruida is limited as none of us is owning such a device. So while this might not the answer you were hoping for, I would recommedn to try it yourself and report issues you found so we can have a look at them.

**Comment by joerlane** [comment]
I own a different Ruida controller. One of the things that makes the Rd6563F different is the controller itself sells for over $1,700. I would be interested to see some pictures of the machine, and the wiring. I looked at the controller itself, and didn't see any driver connection for a galvo head; so I would be rather curious to know how the machine functions.

Even the laser control appears to come from the DB-9 port; which the less expensive fiber lasers do not have available. 

It may still be possible to support assuming the protocol is similar to other Ruida controllers. Sounds like it's close enough that Lightburn "almost" works already. Does the same job always fail at the exact same point with Lightburn?

Would certainly take some effort, but I do enjoy digging into the deep end of the control protocols.

**Reply by JacobWeinrich** [reply]
So what I did is I converted a 5'x10' Plasma table into a 1000w laser cutter
I bought a laser rust cleaner like you see in those Chinese videos using and I swapped the head for a cutting head
so it has a x and axis with a z gantry to move up and down
there's a sensor that keeps the laser head above the metal but the controller requires the z axis be a servo and not a stepper stupid design don't suggest buying that controller there's probbley better ones I just don't got the budget to replace it

on the topic of light burn it works but ends up failed about have way through a big cut (multiple shapes and lines) small stuff works fine though says incompatible file so iv bin looking for a way to make it work.
Lightburn doesn't support the z axis which is fine for me
